msg/CMakeLists.txt

```diff
@@ -161,6 +161,7 @@ set(msg_files
 	RcChannels.msg
 	RcParameterMap.msg
 	Rpm.msg
+	ActuatorControls.msg
```

Comment:
Is this intentional?
```suggestion
```

Reply:
It was a leftover dependency which we removed in a cleanup thanks for pointing out :+1: